src/mocks/films.js

```diff
@@ -17,21 +20,56 @@ const getPosterSrc = (title) => {
   return `img/${src}.jpg`;
 };
 
+const getRatingValue = (rating) => {
+  if (rating === 10) {
+    return `Awesome`;
+  }
+  if (rating < 10 && rating >= 8) {
+    return `Very good`;
+  }
+  if (rating < 8 && rating >= 5) {
+    return `Good`;
+  }
+  if (rating < 5 && rating >= 3) {
+    return `Normal`;
+  }
+  return `Bad`;
+};
+
+const generateComment = () => {
+  return {
+    message: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
+    rating: getRandomInteger(10),
+    author: NAMES[getRandomInteger(NAMES.length)],
+    date: `November 18, 2015`,
+  };
+};
```

Comment:
```suggestion
const generateComment = () => ({
    message: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
    rating: getRandomInteger(10),
    author: NAMES[getRandomInteger(NAMES.length)],
    date: `November 18, 2015`,
  });
```

ну и отступы поправить.